pkg/constants/constants.go

```diff
@@ -76,6 +76,13 @@ const (
 	// DefaultcuratedPackagesRegistry is a containerd compatible registry format that matches all AWS regions.
 	DefaultCuratedPackagesRegistry = "783794618700.dkr.ecr.*.amazonaws.com"
 
+	// KubernetesVersionRegex matches the Kubernetes version field in the cluster config.
+	KubernetesVersionRegex = `(?m)(.*kubernetesVersion:\s*)['"]?(1\.[0-9]{2,})['"]?(.*)$`
+
+	// QuoteRegexReplacement is the target replacement pattern which will add quotes around the Kubernetes version
+	// in the cluster config.
+	QuotedKubernetesVersionRegexReplacement = `${1}"${2}"${3}`
```

Comment:
Do these need to be saved as a variable if we decide to move it to a method where it's only reference in one place?